package.json

```diff
@@ -24,7 +24,7 @@
     "taskcluster-lib-docs": "^10.0.0",
     "taskcluster-lib-loader": "^10.0.1",
     "taskcluster-lib-monitor": "^10.0.0",
-    "taskcluster-lib-validate": "^11.0.2",
+    "taskcluster-lib-validate": "^12.0.0",
```

Comment:
So just to make sure: in `tc-lib-validate` repo, I see a PR merged _after_ this tag. So that change won't get here. Is that according to the master plan?

Reply:
Yes, that's just a change to the tests.

Reply:
Looked closer - that was a change to the test. Ok, you shall pass after all 😂